article/app/controllers/ArticleController.scala

```diff
@@ -29,7 +29,7 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
   }
 
 
-  private def isSupported(c: ApiContent) = c.isArticle || c.isLiveBlog || c.isSudoku
```

Comment:
Nooooo, this will break older Sudoku that were tagged with Sudoku and not Article. Like this one...
![sudoku](https://cloud.githubusercontent.com/assets/76709/6330389/b4cd70ce-bb6f-11e4-8b4d-3d4a3e071395.png)